src/rebar_pkg_resource.erl

```diff
@@ -120,3 +121,62 @@ etag(Path) ->
         {error, _} ->
             false
     end.
+
+ssl_opts(Url) ->
+    case check_ssl_version() of
+        true ->
+            {ok, {_, _, Hostname, _, _, _}} = http_uri:parse(ec_cnv:to_list(Url)),
+            VerifyFun = {fun ssl_verify_hostname:verify_fun/3, [{check_hostname, Hostname}]},
+            CACerts = cacerts(),
+            [{verify, verify_peer}, {depth, 2}, {cacerts, CACerts}
```

Comment:
It is higher than the default (1). I copied it from hex so @ericmj can speak to the choice.